code/src/vs/workbench/api/node/proxyResolver.ts

```diff
@@ -71,7 +71,7 @@ export function connectProxyResolver(
 			}
 		},
 		proxyResolveTelemetry: () => { },
-		useHostProxy: isUseHostProxyEnabled(),
+		isUseHostProxyEnabled,
```

Comment:
Should we add a rebase rule for this file?

Reply:
we not override this part, 
we align this part with the upstream https://github.com/microsoft/vscode/blob/main/src/vs/workbench/api/node/proxyResolver.ts#L74